virttest/utils_test/libvirt.py

```diff
@@ -2972,7 +2972,12 @@ def create_local_disk(disk_type, path=None,
         pass
     cmd = ""
     if disk_type == "file":
-        cmd = "qemu-img create -f %s %s %s" % (disk_format, path, size)
+        if luks is None:
+            cmd = "qemu-img create -f %s %s %s" % (disk_format, path, size)
+        else:
+            cmd = "qemu-img create -f %s "\
+                  "--object secret,id=sec0,data=`printf '%s' | base64`,format=base64 "\
```

Comment:
It is better not to hard code these options here.
I suggest that we add an extra param like:
"qemu-img create -f %s %s %s %s" % (disk_format, extra, path, size)
And you can pass extra with ''--object secret,id=sec..." through calling function.